src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
+    public static void printList(String[] tasks, int taskCount, int[] taskDone) {
+        String[] currList = Arrays.copyOf(tasks, taskCount + 1);
+        System.out.println("____________________________________________________________");
+        System.out.println("PENDING HIT LIST:");
+        for (int i = 0; i < currList.length; i++) {
+            if (currList[i] != null) {
+                if (taskDone[i] == 1) {
+                    System.out.println(i + 1 + "." + " [x] " + currList[i]);
+                }
+                else {
+                    System.out.println(i + 1 + "." + " [ ] " + currList[i]);
+                }
+            }
+        }
+        System.out.println("____________________________________________________________");
+
+    }
+
+    public static int doneTask(String line, String[] tasks) {
+        String[] number = line.split(" ");
+        int taskNumber = Integer.parseInt(number[1]);
+        System.out.println("TARGET NEUTRALISED: " + taskNumber + "." + " [x] " + tasks[taskNumber-1]);
+        return taskNumber-1;
+
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("____________________________________________________________");
+        System.out.println("[The Templar:]");
+        System.out.println("Greetings, Hero. Meet Vector Unit 202 - codename DUKE.\n" + "The state of the art AI assassin you requested. Give DUKE a task, and it shall be done.");
```

Comment:
Maybe text stored in String variables can be created as CONSTANTS (with meaningful name) since they are not changing and placed outside of the main function to improve code clarity.